### ISSUE TITLE
[5-dev] 'Routing' page issue - order of import instructions

### ISSUE THREAD

**Issue by diliondani** [dart-lang/site-webdev#1388]
From URL: https://webdev.dartlang.org/angular/tutorial/toh-pt5#add-the-mini-detail
Version: 5.0.0-alpha+7 (preview)
When adding the method stub to HeroesComponent, we need to also add the import for the angular_router. Otherwise we get the following error: 
```console
[SEVERE] build_web_compilers|ddc on lib/src/heroes_component.module:
Error compiling dartdevc module:angular_tour_of_heroes|lib/src/heroes_component.ddc.js

[error] The name 'NavigationResult' isn't a type so it can't be used as a type argument.
(package:angular_tour_of_heroes/src/heroes_component.dart, line 31, col 10)
```
The instruction to add the import only comes in the next section: 
https://webdev-dartlang-org-dev.firebaseapp.com/angular/tutorial/toh-pt5#update-the-heroescomponent-class

**Comment by chalin** [comment]
Sorry for the confusion, but the router guide isn't ready yet. I'll be replacing the content of those pages with a warning (#1389). In the meantime, see the Tutorial router page.

**Comment by chalin** [comment]
Oops, I read too quickly and thought that you were referring to the router guide. You are right about that toh-5 error; it is due to a recent change to the tutorial (specifically, adding the `NavigationResult` type parameter).

**Comment by chalin** [comment]
Thanks for the issue; it is fixed via #1390.
